pkgs/games/steam/fhsenv.nix

```diff
@@ -163,60 +141,44 @@ in buildFHSUserEnv rec {
     xorg.libxshmfence
     xorg.libXxf86vm
     libelf
-  ] ++ (if (!nativeOnly) then [
-    (steamPackages.steam-runtime-wrapped.override {
-      inherit runtimeOnly;
-    })
-  ] else [
```

Comment:
Have a closer look at this if expression; everything below this are the *packages for nativeOnly*.

This change makes it so nativeOnly packages are always included. That's not what we want!

Reply:
Deleting all of the nativeOnly packages again fixes my issue.

Reply:
you can see how many duplicates we had, more and more launchers were requiring additional dependencies

Reply:
Also, do we not need the runtime? It seems to function without but I wonder why...

Reply:
it's not really used, steam will manage it's own runtimes, the one's packaged within nix are considered bootstrap runtimes, and they are just there to make sure it launches (IIRC)

Reply:
Shouldn't we strive to make nativeOnly a thing then? If all we need is bootstrap for Steam to launch its own runtime, we don't need to package and distribute a huge runtime of Ubuntu packages.

Reply:
It has been broken for years

Reply:
I know but that was before Steam did its own sandboxing.